### ISSUE TITLE
Panics when no classes or empty classes are set as args

### ISSUE THREAD

**Issue by rotmh** [Oudwins/tailwind-merge-go#11]
When the input for `twmerge.Merge` is nothing, or just empty strings, the program panics. 

```
panic: runtime error: index out of range [0] with length 0
...
twmerge/modifier-utils.go:41
...
```

```go
twmerge.Merge("")  // panics
twmerge.Merge()    // panics
```

I don't think it's an expected or wanted behavior, and it can be fixed simply by creating a wrapper for the Merge function, to ensure there is no empty strings, and if there are only empty strings, just return an empty string, e.g.:

```go
var SafeMerge = func(args ...string) string {
	var nonEmptyArgs []string
	for _, arg := range args {
		if arg != "" {
			nonEmptyArgs = append(nonEmptyArgs, arg)
		}
	}
	if len(nonEmptyArgs) == 0 {
		return ""
	}
	return twmerge.Merge(nonEmptyArgs...)
}
```

But I think it would be best to handle it in the package itself.

**Comment by Oudwins** [comment]
Oh wow. This is quite a blunder on my part. I am sorry and thank you so much for reporting it. It will be fixed in a bit. Please update to the latest version

**Comment by Oudwins** [comment]
Closing this now since it has been fixed in #12 update to v0.2.0.

**Comment by rotmh** [comment]
Upgraded and working fine. Thanks!
